source/digital_wallets/index.html.md.erb

```diff
@@ -7,15 +7,21 @@ weight: 2300
 
 # Take a digital wallet payment
 
-Your users can make payments with [Google Pay](https://pay.google.com/intl/en_uk/about/) and [Apple Pay](https://www.apple.com/uk/apple-pay/).
+Your users can make payments with [Google Pay](https://pay.google.com/intl/en_uk/about/) and [Apple Pay](https://www.apple.com/uk/apple-pay/). This page tells you how to enable digital wallet payments, test digital wallet payments, and the restrictions around diigtal wallets.
 
 Users can use digital wallets with:
 
 * payment links
 * payments created through the GOV.UK Pay API
 
+Users can currently only use Google Pay if they're making payments in Google Chrome.
+
+Users can only use Apple Pay if they're making payments in Safari on a recent Apple device.
```

Comment:
Apple Pay does already work in other browsers on iOS 16 (or at least it did in the betas) so “only … Safari” is already out of date: https://9to5mac.com/2022/08/01/apple-pay-web/

Maybe we could say something along the lines of this?

> Users can only use Google Pay if they’re making payments in a supported browser, such as Google Chrome.

> Users can only use Apple Pay if they’re making payments on a compatible Mac, iPhone or iPad in a supported browser, such as Safari.

Bit wordy but maybe you can tighten it a bit.

Reply:
Wordy but I think it works. Pushed the changes, so feel free to re-review @alexbishop1